packages/changesets/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "extends": "@vue-storefront/integrations-tsconfig/tsconfig",
+  "compilerOptions": {
```

Nitpick: Please? 🙏 😄 
```suggestion
{
  "extends": "@vue-storefront/integrations-tsconfig/tsconfig",
  "compilerOptions": {
    "strict": true,
```

---

sure! :D